webpack.config.js

```diff
@@ -404,6 +445,74 @@ function* webpackConfig( env, argv ) {
 			} ),
 		],
 	};
+
+	// Build the Gutenberg entry points.
+	yield {
+		entry: gutenbergEntryPoints,
+		externals: gutenbergExternals,
+		output: {
+			filename:
+				mode === 'production' ? '[name]-[contenthash].js' : '[name].js',
```

Comment:
I see we have many `mode === 'production'` occurrences in this file. How about assigning it to a variable and using the variable instead of repeating this condition every time?

```js
const isProduction = mode === 'production;

...

filename: isProduction ? '[name]-[contenthash].js' : '[name].js',
chunkFilename: isProduction ? '[name]-[chunkhash].js' : '[name].js',

...
```